kipoi/pipeline.py

```diff
@@ -126,3 +126,45 @@ def predict_generator(self, dataloader_kwargs, batch_size=32, **kwargs):
             if i == 0 and not self.dataloader_cls.output_schema.compatible_with_batch(batch):
                 logger.warn("First batch of data is not compatible with the dataloader schema.")
             yield self.model.predict_on_batch(batch['inputs'])
+
+    def input_grad(self, dataloader_kwargs, batch_size=32, filter_idx=None, avg_func=None, layer=None,
```

Comment:
analogous to `predict` and `predict_generator` we should implement two methods:
`input_grad` and `input_grad_generator`